plugins/wasi_nn/ggml.cpp

```diff
@@ -0,0 +1,196 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+#include "ggml.h"
+#include "wasinnenv.h"
+
+#ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_GGML
+#include <common.h>
+#include <llama.h>
+#endif
+
+namespace WasmEdge::Host::WASINN::GGML {
+#ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_GGML
+Expect<ErrNo> load(WasiNNEnvironment &Env, Span<const Span<uint8_t>> Builders,
+                   Device Device, uint32_t &GraphId) noexcept {
+  // The graph builder length must be 1.
+  if (Builders.size() != 1) {
+    spdlog::error("[WASI-NN] Wrong GraphBuilder Length {:d}, expect 1",
```

Comment:
Use `string_literal` for all const strings. This should be `"..."sv`.

Reply:
Updated